subsys/bluetooth/mesh/frnd_cred.c

```diff
@@ -0,0 +1,265 @@
+/*  Bluetooth Mesh */
+
+/*
+ * Copyright (c) 2017 Intel Corporation
+ * Copyright (c) 2020 Lingao Meng
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <string.h>
+#include <errno.h>
+#include <stdbool.h>
+#include <sys/atomic.h>
+#include <sys/util.h>
+#include <sys/byteorder.h>
+
+#include <net/buf.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/mesh.h>
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_MESH_DEBUG_FRND_CRED)
+#define LOG_MODULE_NAME bt_mesh_frnd_cred
+#include "common/log.h"
+
+#include "crypto.h"
+#include "adv.h"
+#include "mesh.h"
+#include "net.h"
+#include "lpn.h"
+#include "friend.h"
```

Comment:
Needs to include frnd_cred.h.

Can remove includes for sys/atomic.h, bluetooth/bluetooth.h, bluetooth/conn.h, adv.h, mesh.h